offer-manager-domain/src/main/groovy/life/qbic/business/offers/OfferContent.groovy

```diff
@@ -0,0 +1,300 @@
+package life.qbic.business.offers
+
+import life.qbic.business.offers.identifier.OfferId
+import life.qbic.datamodel.dtos.business.AcademicTitle
+import life.qbic.datamodel.dtos.business.Affiliation
+import life.qbic.datamodel.dtos.business.Customer
+import life.qbic.datamodel.dtos.business.ProductItem
+import life.qbic.datamodel.dtos.business.ProjectManager
+import life.qbic.datamodel.dtos.projectmanagement.ProjectCode
+
+/**
+ * <h1>A DTO containing the fields required in the offer pdf</h1>
+ *
+ * <p>The content of this class is based on all the fields that need be part in the final offer pdf</p>
+ *
+ * @since 1.1.0
+ *
+*/
+class OfferContent {
+
+    /*Person Information*/
+    /**
+     * The information for the customer for which this offer was created
+     */
+    private String customerFirstName
+    private String customerLastName
+    private String customerTitle
+    /**
+     * The information for the affiliation of the customer selected for this offer
+     */
+    private String customerOrganisation
+    private String customerStreet
+    private String customerPostalCode
+    private String customerCity
+    private String customerCountry
+    /**
+     * The information for the QBiC project manager who was assigned to the project
+     */
+    private String projectManagerFirstName
+    private String projectManagerLastName
+    private String projectManagerTitle
+    private String projectManagerEmail
+
+    /*Project Information*/
+    /**
+     * Date on which the offer was lastly modified
+     */
+    private String creationDate
+    /**
+     * The date on which the offer expires
+     */
+    private String expirationDate
+    /**
+     * The title of the project
+     */
+    private String projectTitle
+    /**
+     * A short objective of the project
+     */
+    private String projectObjective
+    /**
+     * A short description of the experimental design of the project
+     */
+    private String experimentalDesign
+    /**
+     * The identifier for the offer which makes it distinguishable from other offers
+     */
+    private String offerIdentifier
+
+    /*Items*/
+    /**
+     * The items assigned to data generation section of the offer
+     */
+    private List<ProductItem> dataGenerationItems
```

Comment:
I think that should be the OfferItem from #728

Reply:
yes! But it needs to be available so I am waiting for the pr ;)